dist/index.js.LICENSE.txt

```diff
@@ -12,20 +16,11 @@ object-assign
  * @license MIT
  */
 
-/** @license React v17.0.2
+/** @license React v16.13.1
```

Comment:
Weird, bumping down to Node 14, caused this to drop to v 16

Reply:
The lock shows 17, so this is likely a dep that is bringing in v16 on Node 14 so it includes the license.